lab1.js

```diff
@@ -75,7 +76,27 @@ var sentence2 = 'Come over here so you can scratch my belly.';
  HINT: the 'split' method on String will be useful.
 */
 
-// your code goes here
+function splitString(stringToSplit, separator) {
+  var arrayOfStrings = stringToSplit.split(separator);
+  for (var i = 0; i < arrayOfStrings.length; i++) {
+    arrayOfStrings[i] = 'chirp';
+  }
+  // console.log(arrayOfStrings);
+}
+
+var space = ' ';
+
+splitString(sentence1, space);
```

Comment:
if you have a return value from split string, you can assign that value to sentence 1, like this:
sentence1 = splitString(sentence1, space)